contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -470,6 +488,22 @@ def _execute(self, track_name_filter: str, expect_exists: bool = True):
                                  'Track "{}" has no callstacks pane'.format(track.name))
 
 
+class SelectAllCallstacks(CaptureWindowE2ETestCaseBase):
```

Comment:
Also here, please add a class comment. We don't have this consistently everywhere, but I've tried to point it out when I notice it.